ansible/inventories/testnet.yml

```diff
@@ -37,6 +37,7 @@ validators_ejector:
       - "http://{{ (hostvars[groups.ethereum_nodes[0]]).wireguard_address.split('/')[0] }}:5051"
     lido_validator_ejector_operator_id: "{{ lookup('env','VALIDATOR_EJECTOR_OPERATOR_ID') }}"
     lido_validator_ejector_staking_module_id: "{{ lookup('env','VALIDATOR_EJECTOR_STAKING_MODULE_ID') }}"
+    lido_validator_ejector_messages_password: "{{ lookup('env','VALIDATOR_EJECTOR__MESSAGES_PASSWORD') }}"
```

Comment:
Could you replace the double underscore on the single? To match the rest style of the variables writing.
VALIDATOR_EJECTOR__MESSAGES_PASSWORD -> VALIDATOR_EJECTOR_MESSAGES_PASSWORD